service/app/controllers/TelemetryController.java

```diff
@@ -46,6 +46,10 @@
         ProjectLogger.log(
             "TelemetryController:save: Received telemetry in JSON format.", LoggerEnum.INFO.name());
         request.put(Constant.BODY, Json.stringify(request().body().asJson()));
+        ProjectLogger.log(
+            "TelemetryController:save: Telemetry data = "
+                + Json.stringify(request().body().asJson()),
+            LoggerEnum.INFO.name());

```

Comment:
dont log the telemetry request body, there will be huge number of telemetry events and the log file will be full of telemetry events only. please remove this log.

Reply:
ignore this comment as this log statement was removed in the next commit